mmv1/products/alloydb/Instance.yaml

```
@@ -443,6 +443,12 @@ properties:
         type: Boolean
         description: |
           Enabling outbound public ip for the instance.
+      - name: 'allocatedIpRangeOverride'
+        type: String
+        description: |
```

```suggestion
        type: String
        immutable: true
        description: |
```
if the field is immutable, we should add this attribute